src/data/backup.ts

```diff
@@ -288,6 +293,39 @@ export const generateEncryptionKey = () => {
   return result;
 };
 
+export const generateEmergencyKit = (
+  hass: HomeAssistant,
+  encryptionKey: string
+) =>
+  "data:text/plain;charset=utf-8," +
+  encodeURIComponent(`Home Assistant Backup Emergency Kit
+
+This emergency kit contains your backup encryption key. You need this key
+to be able to restore your Home Assistant backups.
+
+Date: ${formatDateTime(new Date(), hass.locale, hass.config)}
+Instance: ${hass.config.location_name}
+URL: ${hass.auth.data.hassUrl}
+Encryption key: ${encryptionKey}
```

Comment:
Put this on a new line so there are no spaces before that could be copied in as part of the key.